parachain/runtime/src/lib.rs

```diff
@@ -873,6 +873,11 @@ impl_runtime_apis! {
             Ok(result.iter().map(|v| (v.0.clone(), BalanceWrapper{amount:v.1})).collect())
         }
 
+        fn get_vaults_with_redeemable_tokens() -> Result<Vec<(AccountId, BalanceWrapper<Balance>)>, DispatchError> {
+            let result = VaultRegistry::get_vaults_with_redeemable_tokens()?;
+            Ok(result.iter().map(|v| (v.0.clone(), BalanceWrapper{amount:v.1})).collect())
```

Comment:
use `into_iter()` instead of `iter()`, that way you get ownership over the items you iterate over, so you won't have to do the clone in `v.0.clone()`.

Not that that this matters a huge deal, but since there are some other comments anyway I figured I'd include this :)